### ISSUE TITLE
Tag latest release

### ISSUE THREAD

**Issue by gusbrs** [magit/with-editor#88]
I may be missing something here, but considering I might have spotted something which may reduce "release friction" I hope you'd also prefer a "false positive" than the omission. So, if I am indeed missing something, feel free to just close and my apologies for the "trigger readiness" in reporting.

Please explain:

1. what behavior you expected

Since commit https://github.com/magit/with-editor/commit/5519b6a67ecd66865b4fdd5447425eee900c54f4 "Release version 3.0.4" has been made almost a day ago, I'd expect this version to be available on Melpa Stable by now. Also because the latest version is required for the released Magit version.

2. what behavior you observed

Melpa Stable has, as of now, v3.0.2 available, not v3.0.4. Also, as of now, the tag for version 3.0.4 does not appear on the "Tags" list of the repository.

3. and how we can reproduce the issue.

Visit https://stable.melpa.org/#/with-editor
And https://github.com/magit/with-editor/tags

**Comment by tarsius** [comment]
I forgot to push those tags. But it turns out that is a good thing because I also did not update the Melpa recipe for `magit` resulting in that package containing `magit-section.el` even though that is distributed as a separate package. The first issue prevents prevents users from installing `magit` and being affected by the second issue. Once the second issue has been addressed, then I will push the tags.
